app/home/routes.py

```diff
@@ -4,11 +4,11 @@
 """
 import os
 from app.home import blueprint
-from flask import render_template, redirect, url_for, request, current_app
+from flask import render_template, redirect, url_for, request, current_app, flash
```

Comment:
I see you are using flash messages 
In case they are not working yet 
You need to set them up in the base html file as well which you are inheriting across.